apps/discovery_api/config/integration.exs

```diff
@@ -49,6 +49,18 @@ config :discovery_api, :elasticsearch,
               type: "keyword",
               index: true
             },
+            keywordFacets: %{
```

Comment:
Did you add these to the runtime.exs as well? If you can find a way to make this suck less for DRYness, please do it :)

Reply:
Have now